provider/terraform/import.rb

```diff
@@ -69,6 +69,14 @@ def import_id_formats(import_format, identity, base_url)
         # Regexes should be unique and ordered from most specific to least specific
         # We sort by number of `/` characters (the standard block separator)
         # followed by number of variables (`{{`) to make `{{name}}` appear last.
+        if id_formats[0].include?('%')
```

Comment:
nit: if you change this to unless, then the body can be something like `id_formats += [short_id_format, etc]` and then the part after the statement would be `id_formats.uniq.reject(etc.)` that way we don't have to repeat this expression in both places